devicetypes/heltun/heltun-tps05-switch.src/heltun-tps05-switch.groovy

```diff
@@ -0,0 +1,680 @@
+/**
+ *  HELTUN TPS05 Switch
+ *
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+metadata {
+	definition (name: "HELTUN TPS05 Switch", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true, mcdSync: true ) {
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Energy Meter"
+		capability "Power Meter"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+
+		fingerprint mfr: "0344", prod: "0004", model: "0003", deviceJoinName: "HELTUN"
```

Comment:
Rename it to `deviceJoinName: "HELTUN Switch"`

Reply:
I would like to keep it only "HELTUN" as this is the name of the main node.
Child devices names are HELTUN SWITCH 1...5 
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/ea6282c658be8d5086e2571f3cd47ff364026379/devicetypes/heltun/heltun-tps05-switch.src/heltun-tps05-switch.groovy#L178

Reply:
`deviceJoinName` must include some type. If `Switch` doesn't work for you, perhaps `Controller`?

Reply:
@PKacprowiczS what about "HELTUN Panel" ?
is that ok ?